sim/pokemon.ts

```diff
@@ -528,7 +542,7 @@ export class Pokemon {
 		return null;
 	}
 
-	allies(): Pokemon[] {
+	allies(faintedToo?: boolean): Pokemon[] {
```

Comment:
I plead guilty !

Reply:
`includeFainted`

Reply:
MacChaeger called it "forTargeting", but I like `includeFainted` more than either.